hazelcast/src/main/java/com/hazelcast/cache/impl/journal/DeserializingEventJournalCacheEvent.java

```diff
@@ -95,4 +95,18 @@ private Data toData(Data data, Object o) {
     public void setHazelcastInstance(HazelcastInstance hazelcastInstance) {
         serializationService = ((SerializationServiceSupport) hazelcastInstance).getSerializationService();
     }
+
+    @Override
```

Comment:
Is it not sufficient to add this to the documentation of the class? Why create additional methods that don't do anything?

Reply:
Checkstyle and/or findbugs. I'll try find a supression.

Reply:
Found the suppression:
```
@SuppressFBWarnings(value = "EQ_DOESNT_OVERRIDE_EQUALS",
        justification = "equality is checked by serialised data in superclass, not deserialised instances in this class")
```
Removed the dummy implementations.